sdk/automation/azure-mgmt-automation/tests/test_cli_mgmt_automation.py

```diff
@@ -30,10 +30,12 @@ class MgmtAutomationClientTest(AzureMgmtTestCase):
 
     def setUp(self):
         super(MgmtAutomationClientTest, self).setUp()
+        self.re_replacer.register_pattern_pair('"value": ".{64}"', '"value": "FakeValue"')
+        self.re_replacer.register_pattern_pair('"Value":".{88}"', '"Value":"FakeValue"')
```

Comment:
Better  to give an example as annotation to explain how regular expression works. It is hard to understand it.